package.json

```diff
@@ -5,9 +5,6 @@
   "license": "MIT",
   "repository": "avajs/babel-preset-stage-4",
   "author": "Mark Wubben (https://novemberborn.net)",
-  "publishConfig": {
-    "access": "public"
-  },
```

Comment:
Why was this removed? https://github.com/sindresorhus/np#public-scoped-packages

Reply:
That's only necessary when the package is first published. Afterwards npm knows its public.

Reply:
Oh, I never realized. I've been putting this config in many packages.